client/components/DrawingGame.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import * as handTrack from 'handtrackjs';
+
+const modelParams = {
+  flipHorizontal: true,
+  // reduce input image size for gains in speed.
+  imageScaleFactor: 0.7,
+  // maximum number of boxes to detect
+  maxNumBoxes: 1,
+  iouThreshold: 0.5,
+  scoreThreshold: 0.75,
+};
+
+let model = null;
+const video = document.getElementById('myVideo');
+const canvas = document.getElementById('canvas');
+const context = canvas.getContext('2d');
+
+export default class DrawingGame extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isVideo: false,
+      message: '',
+    };
+    this.startVideo = this.startVideo.bind(this);
+    // this.toggleVideo = this.toggleVideo.bind(this);
+    this.runDetection = this.runDetection.bind(this);
+    this.startGame = this.startGame.bind(this);
+  }
+
+  startVideo() {
+    handTrack.startVideo(video)
+      .then((status) => {
+        console.log('video started', status);
+        if (status) {
+          this.setState({ isVideo: true, message: 'Video started. Now tracking.' });
+          console.log('video started again...');
+          this.runDetection();
+        } else {
+          console.log('please enable video');
+          this.setState({ message: 'Please enable video.' });
+        }
+      });
+  }
+
+  // toggleVideo() {
+  //   const { isVideo } = this.state;
+  //   if (!isVideo) {
+  //     this.setState({ message: 'Starting video.' });
+  //     this.startVideo();
+  //   } else {
+  //     this.setState({ isVideo: false, message: 'Video stopped' });
+  //     handTrack.stopVideo(video);
+  //   }
+  // }
+
+  runDetection() {
+    const { isVideo } = this.state;
+    const { runDetection } = this;
+    if (model) {
+      model.detect(video).then((predictions) => {
+        console.log('model detected');
+        if (predictions[0]) {
+          // model.renderPredictions(predictions, canvas, context, video);
```

Comment:
do we need to keep these commented-out lines in case we have to use them at some point, or can they be removed?

Reply:
the model.renderPredictions is how we would display a video where the user can see where their hand is located / being tracked. I want to leave this for now as well. Most of the commented out aspects are things we may still use.